kview/src/main/java/dev/ikm/komet/kview/mvvm/view/navigation/ConceptPatternNavController.java

```diff
@@ -246,6 +217,18 @@ public void loadAllPatterns() {
                     if (!hasChildren) {
                         patternNavEntryController.disableInstancesListView();
                     }
+
+                    // add listener for double click to summon the pattern into the journal view
+                    patternHBox.setOnMouseClicked(mouseEvent -> {
+                        // double left click creates the concept window
+                        if (mouseEvent.getButton().equals(MouseButton.PRIMARY)) {
+                            if (mouseEvent.getClickCount() == 2) {
+                                EvtBusFactory.getDefaultEvtBus().publish(JOURNAL_TOPIC, new MakePatternWindowEvent(this,
```

Comment:
If I have 2 Journal windows open then I do not want to open the patterns in both the Journal windows, it should open only on the one I clicked. Using JOURNAL_TOPIC will cause the pattern summon window to open on both the Journal windows. This needs to be changed.

Reply:
Journal_topic is global to the application and should not be used in this scenario. Each Journal Window has a unique topic created. This can be passed into summoned windows. Because of the older APIs we used KometNodeFactory(s) to construct windows before displaying them and we pass the journal window's unique topic uuid into the view properties or KometPreferences object (this was prior to the mvvm and not sure if the topic is passed into the view model).

Reply:
fixed